### PR TITLE
Add test for python code that unlocks locks for dead Nomad jobs.

### PR DESCRIPTION
**Pull request by Talina06**

### CODE REVIEW

File: dispatcher/cleanup.go [hunk 1]
```diff
@@ -82,10 +116,17 @@ job "{{ job_id }}" {
       driver = "raw_exec"
 
 		config {
-			command = "bash"
-			args = ["-exc", "%v"]
+			command = "python"
```
Comment (mohandutt134) [inline dispatcher/cleanup.go]: Either you have to give absolute path or run this inside bash  like `bash -c 'python scripts/cleanup.py'`

**Comment by meson10** [comment]
why wrap it inside bash? Just spin off python directly?

**Comment by Talina06** [comment]
Not wrapping inside bash, review the PR again, @mohandutt134

File: dispatcher/cleanup.go [hunk 2]
```diff
@@ -13,21 +13,53 @@ const CLEANUP_PYTHON = `
...
-        print("Fetching Nomad Job details for Nomad Job Name: " + k + "-" + v)
-
-        status = json.loads(urllib2.urlopen("http://%v/v1/job/" + k + "-" + v).read().decode('utf-8'))['Status']
-        if status == 'dead':
-            print("Deleting lock from Consul for dead Nomad Job: " + key)
-			opener = urllib2.build_opener(urllib2.HTTPHandler)
-            request = urllib2.Request("http://%v/v1/kv/" + key, data='')
-            request.get_method = lambda: 'DELETE'
+consul_url = "http://%v/v1/kv/"
+nomad_url = "%v/v1/job/"
```
Comment (meson10) [inline dispatcher/cleanup.go]: This is against our convention

**Comment by Talina06** [comment]
Review again @meson10

**Comment by Talina06** [comment]
In the same PR : https://github.com/tsocial/tessellate/pull/37/files#diff-e143141fa04412503dc65c2c4dc1dde9

**Pull request closed without merging**